website_sale_hide_discount/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+==========================
+Website Sale Hide Discount
+==========================
+
+Only show the pricelist price in website sale, instead of showing the original
+price crossed out.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/113/50
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/e-commerce/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
```

Comment:
Really? We're not including the source file though & this is included in the template. IIRC there was even a conversation specifically mentioning that this credit was here, so the `svg` wasn't a required inclusion. 

@pedrobaeza - you were involved in this convo IIRC, do you remember the specifics? Am I mis-remembering?